### PR TITLE
[fr] Follows #4912

### PR DESCRIPTION
**Pull request by HavokInspiration**
I rewrote a middleware description as it was probably a copy/paste of the one above in the summary of the available middlewares.

Follows #4912.

### CODE REVIEW

File: fr/controllers/request-response.rst [hunk 1]
```diff
@@ -444,6 +444,31 @@ Vérifier si une langue spécifique est acceptée::
 
     $acceptsFrench = $this->request->acceptLanguage('fr-fr');
 
+.. _request-cookies:
+
+Cookies
+-------
+
+Les cookies de la Request peuvent être lu à travers plusieurs méthodes::
```
Comment (antograssiot) [inline fr/controllers/request-response.rst]: lus ?

File: fr/controllers/request-response.rst [hunk 2]
```diff
@@ -964,6 +1018,107 @@ un code fonctionnel, vous devrez écrire::
...
 
 
+.. php:namespace:: Cake\Http\Cookie
+
+CookieCollections
+=================
+
+.. php:class:: CookieCollection
+
+Les objets ``CookieCollection`` sont accessible depuis les objets Request et
```
Comment (antograssiot) [inline fr/controllers/request-response.rst]: accessibles

File: fr/controllers/request-response.rst [hunk 3]
```diff
@@ -964,6 +1018,107 @@ un code fonctionnel, vous devrez écrire::
...
+des patterns immutable, ce qui permet au caractère immutable des Request et des
+Response d'être préservé.
+
+.. _creating-cookies:
+
+Créer des Cookies
+-----------------
+
+Les objets ``Cookie`` peuvent être définis via le constructor ou en utilisant
+l'interface fluide qui suit les patterns immutable::
```
Comment (antograssiot) [inline fr/controllers/request-response.rst]: Immutables ?

File: fr/controllers/request-response.rst [hunk 4]
```diff
@@ -964,6 +1018,107 @@ un code fonctionnel, vous devrez écrire::
...
+.. php:namespace:: Cake\Http\Cookie
+
+CookieCollections
+=================
+
+.. php:class:: CookieCollection
+
+Les objets ``CookieCollection`` sont accessible depuis les objets Request et
+Response. Ils vous permettent d'intéragir avec des groupes de cookies en utilisant
+des patterns immutable, ce qui permet au caractère immutable des Request et des
```
Comment (antograssiot) [inline fr/controllers/request-response.rst]: Immutables ?

**Comment by HavokInspiration** [comment]
@antograssiot Everything has been fixed

**Comment by antograssiot** [comment]
Nice work Yves

**Pull request merged**